code/style.css

```diff
@@ -59,13 +66,72 @@ select {
   align-content: center;
 }
 
+.count-guess {
+  padding-bottom: 40px;
+  color: #fff;
+  font-weight: 600;
+}
+
+.playlist {
+  margin-bottom: 20px;
+  margin-left: 25px;
+  background-color: #07121aaf;
+  border-radius: 20px;
+  text-align: center;
+  padding: 10px;
+}
+
+.playlist h1 {
+  font-size: 20px;
+  line-height: 20px;
+
+}
+
+.playlist span {
+  font-weight: 800;
+}
+
+iframe {
+  margin: 7px;
+  border: 5px solid #C9B446;
+  border-style: double;
+}
+
 /****** CARD ******/
+
+.flip-card {
+  background-color: transparent;
+  perspective: 1000px;
+}
+
+.flip-card-inner {
+  position: relative;
+  transition: transform 1.2s;
+  transform-style: preserve-3d;
+}
+
+.flip-card:hover .flip-card-inner {
+  transform: rotateY(180deg);
+}
+
+.flip-card-front, .flip-card-back {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  -webkit-backface-visibility: hidden;
+  backface-visibility: hidden;
+}
+
+.flip-card-back {
+  transform: rotateY(180deg);
+}
+
```

Comment:
Sweet css styling!

Reply:
Thank you! Found a great source on w3 schools: [https://www.w3schools.com/howto/howto_css_flip_card.asp]